diff --git a/build/.moduleignore b/build/.moduleignore
index ed36151..022d6ed 100644
--- a/build/.moduleignore
+++ b/build/.moduleignore
@@ -82,7 +82,7 @@ native-is-elevated/deps/**
 
-@vscode/vsce-sign/**
-!@vscode/vsce-sign/src/main.d.ts
-!@vscode/vsce-sign/src/main.js
-!@vscode/vsce-sign/package.json
-!@vscode/vsce-sign/bin/**
+@vscodium/vsce-sign/**
+!@vscodium/vsce-sign/src/main.d.ts
+!@vscodium/vsce-sign/src/main.js
+!@vscodium/vsce-sign/package.json
+!@vscodium/vsce-sign/bin/**
 
diff --git a/build/gulpfile.vscode.ts b/build/gulpfile.vscode.ts
index 187726c..076f08b 100644
--- a/build/gulpfile.vscode.ts
+++ b/build/gulpfile.vscode.ts
@@ -449,3 +449,3 @@ function packageTask(platform: string, arch: string, sourceFolderName: string, d
 				'**/*.wasm',
-				'**/@vscode/vsce-sign/bin/*',
+				'**/@vscodium/vsce-sign/bin/*',
 			], [
diff --git a/build/lib/extensions.ts b/build/lib/extensions.ts
index aacf25c..c4b0391 100644
--- a/build/lib/extensions.ts
+++ b/build/lib/extensions.ts
@@ -116,3 +116,3 @@ export function typeCheckExtensionStream(extensionPath: string, forWeb: boolean)
 function fromLocalNormal(extensionPath: string): Stream {
-	const vsce = require('@vscode/vsce') as typeof import('@vscode/vsce');
+	const vsce = require('@vscodium/vsce') as typeof import('@vscodium/vsce');
 	const result = es.through();
@@ -138,3 +138,3 @@ function fromLocalNormal(extensionPath: string): Stream {
 function fromLocalEsbuild(extensionPath: string, esbuildConfigFileName: string): Stream {
-	const vsce = require('@vscode/vsce') as typeof import('@vscode/vsce');
+	const vsce = require('@vscodium/vsce') as typeof import('@vscodium/vsce');
 	const result = es.through();
diff --git a/build/package.json b/build/package.json
index 8a65120..a36d5c4 100644
--- a/build/package.json
+++ b/build/package.json
@@ -45,3 +45,3 @@
     "@vscode/ripgrep": "^1.17.1",
-    "@vscode/vsce": "3.6.1",
+    "@vscodium/vsce": "3.6.1-258428",
     "ansi-colors": "^3.2.3",
diff --git a/src/vs/platform/extensionManagement/node/extensionSignatureVerificationService.ts b/src/vs/platform/extensionManagement/node/extensionSignatureVerificationService.ts
index 98535c5..cc37d7b 100644
--- a/src/vs/platform/extensionManagement/node/extensionSignatureVerificationService.ts
+++ b/src/vs/platform/extensionManagement/node/extensionSignatureVerificationService.ts
@@ -69,3 +69,3 @@ export class ExtensionSignatureVerificationService implements IExtensionSignatur
 	private async resolveVsceSign(): Promise<typeof vsceSign> {
-		const mod = '@vscode/vsce-sign';
+		const mod = '@vscodium/vsce-sign';
 		return import(mod);
