diff --git i/src/vs/workbench/workbench.common.main.ts w/src/vs/workbench/workbench.common.main.ts
--- i/src/vs/workbench/workbench.common.main.ts
+++ w/src/vs/workbench/workbench.common.main.ts
@@ -223,8 +223,7 @@
 import './contrib/search/browser/search.contribution.js';
 import './contrib/search/browser/searchView.js';
 
-// Search Editor
-import './contrib/searchEditor/browser/searchEditor.contribution.js';
+// Search Editor (stripped by !!APP_NAME_LC!!)
 
 // Sash
 import './contrib/sash/browser/sash.contribution.js';
@@ -235,31 +234,19 @@
 // SCM
 import './contrib/scm/browser/scm.contribution.js';
 
-// Debug
-import './contrib/debug/browser/debug.contribution.js';
-import './contrib/debug/browser/debugEditorContribution.js';
-import './contrib/debug/browser/breakpointEditorContribution.js';
-import './contrib/debug/browser/callStackEditorContribution.js';
-import './contrib/debug/browser/repl.js';
-import './contrib/debug/browser/debugViewlet.js';
+// Debug (stripped by !!APP_NAME_LC!!)
 
 // Markers
 import './contrib/markers/browser/markers.contribution.js';
 
-// Process Explorer
-import './contrib/processExplorer/browser/processExplorer.contribution.js';
-
-// Merge Editor
-import './contrib/mergeEditor/browser/mergeEditor.contribution.js';
-
-// Multi Diff Editor
-import './contrib/multiDiffEditor/browser/multiDiffEditor.contribution.js';
+// Process Explorer (stripped by !!APP_NAME_LC!!)
+// Merge Editor (stripped by !!APP_NAME_LC!!)
+// Multi Diff Editor (stripped by !!APP_NAME_LC!!)
 
 // Commands
 import './contrib/commands/common/commands.contribution.js';
 
-// Comments
-import './contrib/comments/browser/comments.contribution.js';
+// Comments (stripped by !!APP_NAME_LC!!)
 
 // URL Support
 import './contrib/url/browser/url.contribution.js';
@@ -284,8 +271,7 @@
 // Terminal
 import './contrib/terminal/terminal.all.js';
 
-// External terminal
-import './contrib/externalTerminal/browser/externalTerminal.contribution.js';
+// External terminal (stripped by !!APP_NAME_LC!!)
 
 // Relauncher
 import './contrib/relauncher/browser/relauncher.contribution.js';
@@ -361,11 +347,8 @@
 // Code Actions
 import './contrib/codeActions/browser/codeActions.contribution.js';
 
-// Timeline
-import './contrib/timeline/browser/timeline.contribution.js';
-
-// Local History
-import './contrib/localHistory/browser/localHistory.contribution.js';
+// Timeline (stripped by !!APP_NAME_LC!!)
+// Local History (stripped by !!APP_NAME_LC!!)
 
 // Workspace
 import './contrib/workspace/browser/workspace.contribution.js';
