diff --git i/src/vs/workbench/workbench.common.main.ts w/src/vs/workbench/workbench.common.main.ts
--- i/src/vs/workbench/workbench.common.main.ts
+++ w/src/vs/workbench/workbench.common.main.ts
@@ -232,8 +232,7 @@
 import './contrib/search/browser/search.contribution.js';
 import './contrib/search/browser/searchView.js';

-// Search Editor
-import './contrib/searchEditor/browser/searchEditor.contribution.js';
+// Search Editor (stripped by !!APP_NAME_LC!!)

 // Sash
 import './contrib/sash/browser/sash.contribution.js';
@@ -256,14 +255,9 @@
 // Markers
 import './contrib/markers/browser/markers.contribution.js';

-// Process Explorer
-import './contrib/processExplorer/browser/processExplorer.contribution.js';
-
-// Merge Editor
-import './contrib/mergeEditor/browser/mergeEditor.contribution.js';
-
-// Multi Diff Editor
-import './contrib/multiDiffEditor/browser/multiDiffEditor.contribution.js';
+// Process Explorer (stripped by !!APP_NAME_LC!!)
+// Merge Editor (stripped by !!APP_NAME_LC!!)
+// Multi Diff Editor (stripped by !!APP_NAME_LC!!)

 // Commands
 import './contrib/commands/common/commands.contribution.js';
@@ -294,8 +288,7 @@
 // Terminal
 import './contrib/terminal/terminal.all.js';

-// External terminal
-import './contrib/externalTerminal/browser/externalTerminal.contribution.js';
+// External terminal (stripped by !!APP_NAME_LC!!)

 // Relauncher
 import './contrib/relauncher/browser/relauncher.contribution.js';
@@ -371,11 +364,10 @@
 // Code Actions
 import './contrib/codeActions/browser/codeActions.contribution.js';

 // Timeline
 import './contrib/timeline/browser/timeline.contribution.js';

-// Local History
-import './contrib/localHistory/browser/localHistory.contribution.js';
+// Local History (stripped by !!APP_NAME_LC!!)

 // Workspace
 import './contrib/workspace/browser/workspace.contribution.js';
